src/core/graphql/driver.ts

```
@@ -1,19 +1,43 @@
+  /** Cannot be `request` as {@link import('graphql-yoga').YogaInitialContext.request} overrides it */
+  req: IRequest;
+  response: IResponse;
+}
+
+export type DriverConfig = GqlModuleOptions &
+  Omit<YogaServerOptions<ServerContext, GqlContextType>, 'context' | 'schema'>;
+
 @Injectable()
-export class Driver extends YogaDriver<'fastify'> {
+export class Driver extends AbstractDriver<DriverConfig> {
+  private yoga: YogaServerInstance<ServerContext, {}>;
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)